data-sources/pihole/config.yml

```diff
@@ -0,0 +1,22 @@
+id: pihole
+displayName: PiHole
+description: |
+A [PiHole](https://pihole.net) is a free DNS server utility that runs on a Raspberry Pi (or other inexpensive device) and captures DNS requests before they're sent out of the network, filtering discrete elements that are spam, harmful, or wasteful. The result is a faster and safer network. 
+
+This quickstart monitors the PiHole by leveraging it's built-in API to display information about the type and quantify of network requests being handled. 
+
+This quickstart requires the pihole-detailed.yml file found at [https://github.com/newrelic/nri-flex/tree/master/examples](https://github.com/newrelic/nri-flex/tree/master/examples).
+
+icon: logo.svg
```

Comment:
I need to take a look into why this line is causing a linting error. In the meantime, I _did_ notice that the SVG file supplied is not valid (Github can't even render it). Would you be willing to update and submit a new version of that logo?

Reply:
It looks like the icon provided for the quickstart is valid, so perhaps we can use that one (if the content is the same).

Reply:
Ah, I believe the linting error is due to your multi-line description above _not_ being indented in. The linter thinks that entire bit of text is the key. If you update that, this should be happy.

Reply:
I didn't include a logo at all, and presumed a default would be used (I'm a NR employee, so I didn't think a logo was needed. 

I'll update the text. Just a sec...

Reply:
OK, updated. I'm VERY new to git/github so if I have to do something besides hit the "resolve conversation" button below, lemme know. Either here or in slack.